src/sassAutocomplete.ts

```diff
@@ -170,13 +211,15 @@ class SassCompletion implements CompletionItemProvider {
 
     let atRules = [],
       properties = [],
-      values = [];
+      values = [],
+      variables = [];
 
     if (value) {
       values = getValues(cssSchema, currentWord);
     } else {
       atRules = getAtRules(cssSchema, currentWord);
       properties = getProperties(cssSchema, currentWord, config.get('useSeparator', true));
+      variables = getVariables();
```

Comment:
I suggest that this should be called outside of the `if`. Since variables can be used in multiple scopes they should be available everywhere. Calling `getVariables()` immediately when `variables` is defined allows this.

When i tried with it In the current position it won't complete variables in css values.

![vscode-sass-current](https://cloud.githubusercontent.com/assets/2156395/19690698/6b8856b4-9ac9-11e6-8d29-17ff00e07302.gif)